alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -265,8 +265,14 @@ feesOK ::
     HasField "_minfeeA" (Core.PParams era) Natural,
     HasField "_minfeeB" (Core.PParams era) Natural,
     HasField "_prices" (Core.PParams era) Prices,
-    HasField "_collateralPercentage" (Core.PParams era) Natural,
-    HasField "address" (Alonzo.TxOut era) (Addr (Crypto era))
+    {- Core.Script era ~ Script era,
+    Core.TxBody era ~ TxBody era,
+    Show (Core.Value era),
+    PrettyA (Core.Value era),
+    PrettyA (Core.PParamsDelta era),
+    PrettyA (Core.AuxiliaryData era), -}
```

Comment:
I guess we are leaving these constraints as comments here since they are helpful for debugging?

Reply:
I removed them